### ISSUE TITLE
Requesting for Filipino translations

### ISSUE THREAD

**Issue by vinzruzell** [mit-cml/appinventor-sources#1151]
I would like to ask if you accept filipino translations.

**Comment by ewpatton** [comment]
@vinzruzell Yes we accept translations. If you have experience using git and GitHub, you can submit a pull request. If not, please [send me an email](mailto:ewpatton@mit.edu) with a Google account and I will share a semiautomated translation with you.

**Comment by vinzruzell** [comment]
Okay sir. I WILL submit a pull request.
By the way, can I ask what is the.content of automated translations sir?

**Comment by ewpatton** [comment]
@vinzruzell We have been exploring the use of Google Translate to bootstrap the translations into a given language using machine learning. There is a service where we can share the partially translated messages file, but it requires a Google account to use (i.e., there isn't an option to publicly share the document).

**Comment by vinzruzell** [comment]
Am okay sir. Since I am interested in contributing filipino translation thru github as per utopian rules. I would like to ask if your open for the contribution?

**Comment by ewpatton** [comment]
Yes. Please submit a pull request against the master branch when you have finished your translation.

**Comment by vinzruzell** [comment]
This is my email sir for you to s3nd the file. vrosell2015@gmail.com

**Comment by vinzruzell** [comment]
For me to have the file needed for translation. Thank you.

**Comment by vinzruzell** [comment]
Hi sir..can you send a github link for the file for translation as per utopian requirement.

**Comment by ewpatton** [comment]
@vinzruzell Please see the [i18n/filipino branch](https://github.com/ewpatton/appinventor-sources/tree/i18n/filipino) in my fork. You will need to translate the two files appengine/src/com/google/appinventor/client/OdeMessages_fil.properties and blocklyeditor/src/msg/fil/_messages.js.

**Comment by vinzruzell** [comment]
Okay sir. Tnx

**Comment by vinzruzell** [comment]
Sir I would like to ask the way of translation here...Am I going to translate the words after equal sign? Or am I going to translate the words with no spaces or not. Pls reply for confirmation. Thank you sir

**Comment by ewpatton** [comment]
You will need to translate all of the strings following the = sign, with the exception of the following:

1. Do not translate the keys that start with "SwitchTo", e.g. "SwitchToEnglish"-- these are kept in the target language so that users can read the language name.
2. Do not translate URLs except in the case of those that reference Wikipedia, in which case if a Wikipedia page exists in the target language you can change the URL to point to the translated page.
3. Strings that map to symbols (e.g., =, >) should remain the same.

For properties/methods/events with multiple words, you should translate them as if they had spaces, but the final string should not contain spaces.

**Comment by vinzruzell** [comment]
hi sir,
I have already made a pull request. I hope You will accept it. Thank you

**Comment by vinzruzell** [comment]
Partial Translation of OdeMessages_fil.properties #1152

**Comment by HenriquesLee** [comment]
Hi, I checked and there are around 19-20 languages available. If you would like it i could try to implement a code that can add various other languages. Please confirm if I should start working on this issue

**Comment by SusanRatiLane** [comment]
> Hi, I checked and there are around 19-20 languages available. If you would like it i could try to implement a code that can add various other languages. Please confirm if I should start working on this issue

This PR is very outdated. Here is the information you need to know about translating App Inventor: https://community.appinventor.mit.edu/t/about-translating-app-inventor/47582/26
